packages/image_picker/image_picker_android/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -630,7 +630,18 @@ public boolean onActivityResult(
 
   private void handleChooseImageResult(int resultCode, Intent data) {
     if (resultCode == Activity.RESULT_OK && data != null) {
-      String path = fileUtils.getPathFromUri(activity, data.getData());
+      Uri uri;
+      // On Android 10 devices using Android Photo Picker, the Uri from getData() could be null.
+      if (data.getData() != null) {
+        uri = data.getData();
+      } else if (data.getClipData() != null && data.getClipData().getItemCount() == 1) {
```

Comment:
The documentation for ClipData.Item `getUri` isn't perfectly clear, but the documentation for other methods (ex [here](https://developer.android.com/reference/android/content/ClipData.Item#getUri())) seems to indicate it can return null. Can you add logic to the check here for that case?

Reply:
Good point, I'm adding it now